py3status/modules/uname.py

```diff
@@ -38,20 +38,14 @@ class Meta:
             ],
         }
 
-    def show_uname(self):
-        system, node, release, version, machine, processor = uname()
-        response = {
+    def uname(self):
+        keys = ['system', 'node', 'release', 'version', 'machine', 'processor']
```

Comment:
please sort them alphabetically

Reply:
I don't think if that's a good idea. We follow the tuples. https://docs.python.org/2/library/platform.html

Reply:
Yeah, that doesn't work... Updated w/ `{release}` only. I saw somebody's screenshot today in IRC... She only had `{release}` too... except that I don't know if she was running `py3status`. If that is not enough... Damn.

Reply:
okay

Reply:
yes I understand, it's because of the zip, no problem, keep this list as is